src/sql/workbench/browser/editor/profiler/dashboardInput.ts

```diff
@@ -84,18 +83,8 @@ export class DashboardInput extends EditorInput {
 			return '';
 		}
 
-		let name = this.connectionProfile.connectionName ? this.connectionProfile.connectionName : this.connectionProfile.serverName;
-		if (this.connectionProfile.databaseName
-			&& !this.isMasterMssql()) {
-			// Only add DB name if this is a non-default, non-master connection
-			name = name + ':' + this.connectionProfile.databaseName;
-		}
-		return name;
-	}
-
-	private isMasterMssql(): boolean {
-		return this.connectionProfile.providerName === mssqlProviderName
-			&& this.connectionProfile.databaseName?.toLowerCase() === 'master';
+		let newTitle = this._connectionService.getEditorConnectionProfileTitle(this.connectionProfile);
```

Comment:
Please add fallback to original title here, dashboard title is empty for normal connections.

Reply:
What do you mean by this? The Dashboard title at the top shows the connection name  on the main branch of ADS

Reply:
Thanks, it's resolved with new changes.